.travis.yml

```diff
@@ -1,9 +1,14 @@
 sudo: false
 language: cpp
 compiler:
-- clang
+  - clang
 os:
-- osx
+  - osx
+before_install:
+  - wget https://raw.githubusercontent.com/google/styleguide/gh-pages/cpplint/cpplint.py
```

Comment:
UpperCamelCase for functions??? are you nuts???
Kidding xD, but seriously, no...
Cheap function named following std convention is clearly an evil thing as it lead to confusion.
On the enum I may agree.
We could keep arguing for a bit a then vote, what do you think?
just to expose points of view.